bitrise.yml

```diff
@@ -313,10 +313,13 @@ workflows:
                 if [[ `uname -s` == "linux" ]]
                 then
                   CACHE_ARCHIVE_URL="https://storage.googleapis.com/cache_pull_test_bucket/linux.tar.gz"
+                  BITRISEIO_STACK_ID="linux-docker-android"
```

Comment:
The stack IDs are hardcoded in the test files, opted to no regenerate the test files as Stack ID can vary.